lib/digital_scale_exception.dart

```diff
@@ -0,0 +1,10 @@
+import 'dart:io';
+
+class DigitalScaleException extends IOException {
```

Comment:
How about defining more specific exception types? This way, the application can have different handling mechanisms for each type. @luiges90

Reply:
there are only one kind of exception at the moment, but I can add a base class I guess.